components/style/values/computed/image.rs

```diff
@@ -88,7 +90,10 @@ impl GenericLineDirection for LineDirection {
                 if compat_mode != CompatMode::Modern => true,
             LineDirection::Corner(..) => false,
             #[cfg(feature = "gecko")]
-            LineDirection::MozPosition(_, _) => false,
+            LineDirection::MozPosition(Some(Position {
+                horizontal: LengthOrPercentage::Percentage(Percentage(x)),
+                vertical: LengthOrPercentage::Percentage(Percentage(y)),
+            }), None) if x == 0.5 && y == 0.0 => true,
```

Comment:
Same here as below. Also, feel free to make this slightly more readable with a nested match if necessary, whatever you feel comfortable with.